pkg/core/database/interfaces.go

```diff
@@ -94,6 +94,8 @@ type Transaction interface {
 
 	FetchOutputExists(destkey []byte) (bool, error)
 
+	FetchOutputUnlockHeight(destkey []byte) (uint64, error)
+
 	SaveBidValues([]byte, []byte) error
```

Comment:
not related to this PR.
Could we use Fetch/Store as prefixes to be aligned with the naming convention for this pkg?

Reply:
Fair point